index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <!-- SECTION TITLE  -->
+      <div class="header">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p id="secondary-heading">The man who saved a billion lives</p>
+      </div>
+      <!-- SECTION IMAGE  -->
+      <div id="img-div">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" id="image" alt="" />
+        <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+      </div>
```

Comment:
Please use semantic element for the container 
```
<section>
...
</section>
```

You can change the div tag to section tag

Reply:
- [ ] Once again, It would be better if you can provide an explanation, a "why" for each requested change as this will help the person making the changes understand the reason & learn from their mistakes. ✔

Reply:
Please use semantic element for the container 
```
<section>
...
</section>
```

You can change the div tag to section tag

## Using the section tag under the main tag is important for several reasons:

1. Accessibility: Screen readers and other assistive technologies can use the section tag to help users navigate a web page. By using semantic tags like main and section, you can make your content more accessible to users with disabilities.

2. Search Engine Optimization (SEO): Search engines like Google use the HTML structure of a web page to understand the content and context of the page. By using semantic tags like main and section, you can help search engines better understand the structure and organization of your content, which can improve your search engine rankings.

3. Readability and maintainability: Using semantic tags like main and section can help make your code more readable and easier to maintain. By using tags that accurately describe the content and structure of your web page, it can be easier for other developers to understand and modify your code.

On the other hand, using the div tag does not provide any semantic meaning to the content and is typically used for general layout and styling purposes. While it is still possible to create a well-structured and accessible web page using only div tags, using semantic tags like main and section can provide additional benefits for accessibility, SEO, and code maintainability.